man/fi_domain.3.md

```diff
@@ -76,11 +76,14 @@ domain interfaces are outside the scope of this documentation.
 
 ## fi_domain_bind
 
-Associates an event queue with the domain.  An event queue bound to a
-domain will be the default EQ associated with asynchronous control
-events that occur on the domain or active endpoints allocated on a
-domain.  This includes CM events.  Endpoints may direct their control
-events to alternate EQs by binding directly with the EQ.
+Associates an event queue with a fid.
+
+Currently, only domains can be associated with an event queue.  An
```

Comment:
I actually wanted it the other way -- that domains are (currently) the only object that can be associated with an event queue.

However, the two statements are not mutually exclusive.  Should I say both things?

* domains are the (currently) only objects that can be bound to event queues
* event queues are (currently) the only objects that can be bound to domains